angular-spa-ui/js/search-app/manage-account-module/manage-account-ctrl.js

```diff
@@ -90,58 +113,99 @@ module.exports = function($scope, $state, accountService, appLocalStorage) {
             firstName: '',
             lastName: ''
         },
-        userEmailNew: '',
+        inptUserEmail: document.getElementById("inpt-user-email"),
         isVisibilityFormUserName: true,
         isVisibilityFormUserEmail: true,
     };
 
-    // Action on the view (Events)
+    /**
+     * Object contains the functions 
+     * that provide actions for the elements in the DOM.
+     */
     var viewAPI = {
 
+        /**
+         * Event on UI - edit
+         * The function to change the user Name
+         */
         editUserName: function() {
-
-console.log(this)
-console.log($scope)
-
-            _optionsAPI.getUserNameParam(model.userData.userName);
-            return model.isVisibilityFormUserName = _optionsAPI.isHideForm;
+            priveteAPI.getUserNameParam(vm.model.userData.userName);
+            return vm.model.isVisibilityFormUserName = !vm.model.isVisibilityFormUserName;
         },
+
+        /**
+         * Event on UI - cancel
+         * Function to cancel the change by user Name
+         * Return form is default state
+         */
```

Comment:
Method name must be obvious. I catched a black screen of death when parsed 'btnCancelUserName' in my head.